pkgs/os-specific/linux/spl/default.nix

```diff
@@ -21,6 +21,11 @@ stdenv.mkDerivation rec {
 
   patches = [ ./install_prefix.patch ];
 
+  # Backported fix for 0.7.13 to build with 5.1, please remove when updating to 0.7.14
+  postPatch = optionalString (versionAtLeast kernel.version "5.1") ''
```

Comment:
Isn't it better to use fetchpatch on https://github.com/zfsonlinux/zfs/commit/b46fd243d57d0af4b4b568d8c0c4b30b79059ecc

Reply:
As mentioned in the commit message:

>This can't be applied cleanly as a patch, since spl has been moved
>into the zfs repo since 0.7.13.

Reply:
Maybe add a comment next to the `version` definition above that this postPatch bit should be removed, too? To help ensure it is found and removed upon update.

Otherwise lgtm.